test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/VsCredentialProviderImporterTests.cs

```diff
@@ -77,65 +75,6 @@ private VsCredentialProviderImporter GetTestableImporter()
             return importer;
         }
 
-        [Fact]
-        public void WhenMultipleProvidersMatchingVstsContractFound_ThenInsertAll()
```

Comment:
You are probably hitting the compile problem because there's an assembly that's supposed to be embedded, but it's not getting embedded.

Usually it's something like: 

Proj A -> Proj B. 

Both depend on AssemblyX.dll which needs to be embedded. 
This can happen if ProjB doesn't embed, but ProjA embeds I think.
So usually the answer to that error would be to add the `embedinterop.targets` to a new project or add a new assembly to that list.

Reply:
I tried adding setting this property all around, and it doesn't work.  I also tried setting the property via reflection and it doesn't work.

NuGet.VisualStudio should be embedded by default anyway.

Reply:
With this change NuGet.PackageManagement.VisualStudio brings in package Microsoft.VisualStudio.SDK.EmbedInteropTypes, which brings in a build target that sets EmbedInteropTypes to true for NuGet.VisualStudio.  That's what I meant when I said "NuGet.VisualStudio should be embedded by default anyway."

Prior to this change, NuGet.PackageManagement.VisualStudio did not bring in that build target, so EmbedInteropTypes was false for NuGet.VisualStudio.  That's how the now deleted tests used to pass.

It may be possible to not embed interop types for NuGet.VisualStudio after this change, but the new default seems correct to me.